packages/proto-rpc/src/index.ts

```diff
@@ -0,0 +1,4 @@
+export { RpcCommunicator, RpcCommunicatorConfig, RpcCommunicatorEvents, StatusCodes } from './RpcCommunicator'
```

Comment:
Rename this to `exports.ts` and remove `browser.ts` (also in `dht` package?)

Reply:
Why? It seems that there is index.ts also in the client package.